ouroboros-consensus/src/Ouroboros/Consensus/BlockchainTime/SlotLength.hs

```diff
@@ -15,11 +15,15 @@ import           Data.Fixed
 import           Data.Time
 import           GHC.Generics (Generic)
 
-import           Cardano.Prelude (NoUnexpectedThunks)
+import           Cardano.Prelude (NoUnexpectedThunks, GShowK1 (..),
```

Comment:
Same comment as in https://github.com/input-output-hk/cardano-base/pull/74/:

Why is it necessary to import `GShowK1 (..)`? It would be nice if we could avoid that.

Reply:
I'm removing the generic stuff I wrote and using `Quiet` instead. Closing this thread as Resolved.